RS.Fritz.Manager.API/Utils/SpecialFolder.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace RS.Fritz.Manager.API
```

Comment:
Can you move this to the UI project? The API is cross platform, while the UI is Windows only.